### PR TITLE
Us1 add book 

### PR DESCRIPTION
**Pull request by alexandregrue**

### CODE REVIEW

**Review by ghost — DISMISSED** [review]

**Review by ghost — DISMISSED** [review]
ok for me. great job !

File: src/Model/AuthorsManager.php [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+
+class AuthorsManager extends AbstractManager
+{
+    public const TABLE = 'author';
+    /* Get element about book to choose in form */
+    /* public function selectAll(): array
+    {
+        $statement = 'SELECT id AS author_id, name AS author_name FROM author;';
+        return $this->pdo->query($statement)->fetchAll(\PDO::FETCH_ASSOC);
+    } */
```
Comment (RomainClair) [inline src/Model/AuthorsManager.php]: There shouldn't be dead code in PRs

File: src/Controller/BooksController.php [hunk 2]
```diff
@@ -0,0 +1,114 @@
...
+
+use App\Model\FormProcessing;
+use App\Model\StatusManager;
+use App\Model\AuthorsManager;
+use App\Model\EditorsManager;
+use App\Model\FormatsManager;
+use App\Model\CategoriesManager;
+use App\Model\LocationsManager;
+
+class BooksController extends AbstractController
```
Comment (RomainClair) [inline src/Controller/BooksController.php]: Having a BookController and a BooksController class is very confusing. You probably should only have one

**Review by RomainClair — CHANGES_REQUESTED** [review]
This PR is not atomic.
You should break it into a : 
- add author
- add editor
- add location, etc...
- and the add book

Furthermore, it's very hard to test because the addBook pages display white buttons on a white background.

You probably should consider hosting you assets (fonts) on your website.
One of the consequence is that your site will not depend on external internet ressources, so it will work during offline demos.

I guess the cover_page images are the result of testing so they should probably be in the .gitignore and not in the git repository

Why are there all the book field int the addAuthor form ?

**Review by Thomas905 — DISMISSED** [review]
good for me modification ask by Romain is OK

**Review by Thomas905 — DISMISSED** [review]
ok for me

**Review by Thomas905 — APPROVED** [review]

**Review by RomainClair — APPROVED** [review]

**Pull request merged**